addons/mail/models/res_partner.py

```diff
@@ -264,7 +264,7 @@ def _to_store_defaults(self, target: Store.Target):
             "avatar_128",
             "im_status",
             "is_company",
-            Store.One("main_user_id", ["share"]),
+            Store.One("main_user_id", ["share"], sudo=True),
```

Comment:
We try to add comments above sudo lines in our code to explain why it is necessary and why it is acceptable.